src/lib.rs

```diff
@@ -4221,6 +4263,161 @@ fn map_darwin_target_from_rust_to_compiler_architecture(target: &Target) -> &str
     }
 }
 
+// Rust and clang/cc don't agree on what equivalent flags should look like either.
+fn rustc_to_cc_flag(flag: &str, family: ToolFamily) -> Option<OsString> {
+    match family {
+        ToolFamily::Clang { .. } | ToolFamily::Gnu => match flag {
```

Comment:
Do we really want it to be a match given that every branch just use `if flag.starts_with`?

Reply:
Up to you really, I don't think it matters too much. I prefer the match style over a ladder of ifs but it'll compile to the same thing I'm pretty sure.

Reply:
Thanks, in that case, I'm ok with the existing style.